tests/src/test/330/scala/com/nvidia/spark/rapids/IntervalSuite.scala

```diff
@@ -68,6 +70,8 @@ class IntervalSuite extends SparkQueryCompareTestSuite {
   }
 
   test("test GPU cache interval when reading/writing parquet") {
+    // temporarily skip the test on Spark 3.3.0-https://github.com/NVIDIA/spark-rapids/issues/5497
+    assume(spark.SPARK_VERSION_SHORT < "3.3.0")
```

Comment:
NIT: can we following the methods in `SparkQueryCompareTestSuite` like `assumePriorToSpark320` .
```
// add in VersionUtils
VersionUtils.isSpark330OrLater

// add in SparkQueryCompareTestSuite
assumePriorToSpark330
```

Reply:
done